lib/parallel.rb

```diff
@@ -123,6 +129,25 @@ def next
       [item, index]
     end
 
+    def runloop
```

Comment:
would this make sense ?
```suggestion
    def run_runloop
```

Reply:
You're right. Please feel free to change anything.